CODEOWNERS

```diff
@@ -1,2 +1,4 @@
 # Default code owner for the repository
 * @mike-petersen-ni
+
+/docs/source/policies/  @amstewart
```

Comment:
ACK? @mike-petersen-ni

Reply:
Are you intending to be the exclusive owner in this directory?

> [Order is important; the last matching pattern takes the most precedence.][1]

[1]: <https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file>

Reply:
Hmm. No; I was expecting that you would be on the reviewer list as well. I suppose I would have to explicitly add your name on the same rule line as well then. Will fix.

Reply:
Fixed Patch V2.